website/project/__init__.py

```diff
@@ -109,6 +109,35 @@ def new_collection(title, user):
 
     return node
 
+def new_public_files_collection(user):
+    """Create a new folder project.
+
+    :param str title: Node title
+    :param User user: User object
+    :return Node: Created node
+
+    """
+    existing_public_files_collections = Node.find(
```

Comment:
This seems like a somewhat unnecessary variable declaration. Consider instead:
```python
if Node.find(<public_files_query>).count():
    raise NodeStateError('<message>')
```
Not a blocker.